service-api/src/main/java/greencity/client/UserRemoteClient.java

```diff
@@ -64,4 +65,12 @@ public interface UserRemoteClient {
      */
     @PutMapping("/user/markUserAsDeactivated")
     void markUserDeactivated(@RequestParam(UUID) String uuid);
+
+    /**
+     * Checks if current user has a password set.
+     *
+     * @return {@link HasPasswordDto} with {@link Boolean} field.
+     */
+    @GetMapping("/ownSecurity/has-password")
```

Comment:
keep some consistency. - /own-security/has-password will be better
or
/own-security/password-status - will be better